### PR TITLE
Adding detailed contribution instructions to readme

### PR DESCRIPTION
**Pull request by JakeBat**
## Description
Updated README to have detailed setup and local testing instructions, removed the under construction warning from top if README, updated travis.yml to correctly use the release profile

## Testing
Manually verified provided instructions (minus JDK and Maven install) on fresh pull of the repo.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -84,12 +82,35 @@ newApplicant.getFirstName();
...
+3. Inside of the top level directory of the project run the following command:
+    ```shell script
+    mvn clean install
+    ```
+   This will install all necessary dependencies and build the model classes used by the project.
+   
+#### Testing
+
+1. Run `mvn test` to run all existing automated tests.
+2. View tests in `src/test/java/com/onfido` for examples of writing tests of your own.
```
Comment (ben-superdevx) [inline README.md]: Can we add `onfido-java/` to the start of the path here?

Comment (JakeBat) [reply README.md]: Updated

File: README.md [hunk 2]
```diff
@@ -84,12 +82,35 @@ newApplicant.getFirstName();
...
+1. Install JDK 1.8 or above, details for which can be found below:
+    - Setup a Oracle account for JDK download: <https://profile.oracle.com/myprofile/account/create-account.jspx>
+    - Download the JDK: <https://www.oracle.com/technetwork/java/javase/downloads/jdk8-downloads-2133151.html>
+    - Follow the install instructions: <https://docs.oracle.com/javase/8/docs/technotes/guides/install/install_overview.html>
+    - Set the JAVA_HOME variable: https://www.baeldung.com/java-home-on-windows-7-8-10-mac-os-x-linux
+2. Install Apache Maven, details for which can be found below:
+    - Download the proper binary archive, `.zip` for Windows and Mac, or `.tar` for Linux: <http://maven.apache.org/download.cgi>
+    - Follow the install instructions: <http://maven.apache.org/install.html>
+3. Inside of the top level directory of the project run the following command:
+    ```shell script
```
Comment (ciaran16) [inline README.md]: I don't think this is a valid language code? Just `sh` works - ```sh

**Comment by JakeBat** [comment]
Will merge after first successful deployment of the master branch

**Pull request merged**